packages/eslint-config-mc-app/package.json

```diff
@@ -22,8 +22,8 @@
   "module": "./index.js",
   "peerDependencies": {
     "babel-eslint": "10.x",
-    "eslint": "5.x || 6.x",
-    "eslint-config-airbnb-base": "13.x",
+    "eslint": "^5.16.0 || ^6.1.0",
+    "eslint-config-airbnb-base": "13.x || 14.x",
```

Comment:
Given the range above I feel we can still support both versions in theory. However v5 and v14 won't work together. Starting to believe we can't really get around a breaking change here?

Reply:
Ah good point...hmm maybe we should plan for a new breaking release, including things like react 16.9 and #801

Reply:
Sounds best. I'll create a `release/15.0.0` and we can point things there?

Reply:
I opened an issue to track those things:
https://github.com/commercetools/merchant-center-application-kit/issues/964

Reply:
> I'll create a release/15.0.0 and we can point things there?

Sounds good, thanks!